modules/whoisquery/whoisquery.go

```diff
@@ -0,0 +1,96 @@
+package whoisquery
+
+import (
+	"errors"
+	"time"
+
+	"github.com/netdata/go-orchestrator/module"
+	"github.com/netdata/go.d.plugin/pkg/web"
+)
+
+func init() {
+	creator := module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery: 60,
+		},
+		Create: func() module.Module { return New() },
+	}
+
+	module.Register("whoisquery", creator)
+}
+
+func New() *WhoisQuery {
+	return &WhoisQuery{
+		Config: Config{
+			Timeout:       web.Duration{Duration: time.Second * 2},
+			DaysUntilWarn: 14,
+			DaysUntilCrit: 7,
```

Comment:
i think we need to tune params, like 3 month warning, 1 month critical. What do you think?